### DISCUSSION TITLE
Disable top-bar when using single toolbar?

### DISCUSSION THREAD

**Discussion by Icy-Thought** [zen-browser/desktop#3933] [Q&A]
I have no use for the window control buttons available present in the top-bar, because of my window manager. From what I can tell, there is no zen-mod available which does this nor is there an option within zen-browser's settings which allows this change. Therefore I thought I should ask if there are any plans to support this option upstream or a zen-mod in the writing which gets rid of the top-bar.

IIRC one could achieve just that by modifying `userchrome.css`, but it does not hurt to ask before DIYing the solution.

**Comment by mr-cheffy** [comment]
New zen release will have `zen.view.experimental-no-window-controls` in about:config

**Reply by Icy-Thought** [reply]
Wonderful! I appreciate the fast response! :)
